docs/guide/key-management.md

```diff
@@ -1,20 +1,170 @@
 # Key Management
 
-Here we explain a bit how to real with your keys, using the `basecli keys` subcommand.
+Here we explain a bit how to work with your keys, using the `basecli keys` subcommand.
 
-**TODO**
+**Note:** This keys tooling is not considered production ready and is for dev only.
 
-## Creating keys
+We'll look at what you can do using the six sub-commands of `basecli keys`:
 
-Create the keys and store a key phrase. No other way to recover it.
+```
+new
+list
+get
+delete
+recover
+update
+```
+
+## Create keys
+
+To get an app started quickly, or for dev/test scripting, the `basecli keys new alice --unsafe` command can be used. However, this tutorial is instead concerned with the advanced features of the tooling that don't compromise safety considerations.
```

Comment:
#213 ... only an idea. I'm not opiniated on this since piping things around does the trick, if you don't want it, close the issue & I'll remove from the docs.